### PR TITLE
Add RouterListToControllerAnnotationsRector

### PR DESCRIPTION
**Pull request by TomasVotruba**
From @entrydo project

### CODE REVIEW

File: packages/NetteToSymfony/src/Rector/RouterListToControllerAnnotationsRector.php [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+
+            $getFqnTypeNode = $staticReturnType->getFqnTypeNode();
+            if ($getFqnTypeNode === null) {
+                return null;
+            }
+
+            if ($this->isName($getFqnTypeNode, $this->routeListClass)) {
+                return null;
+            }
+        }
```
Comment (TomasVotruba) [inline packages/NetteToSymfony/src/Rector/RouterListToControllerAnnotationsRector.php]: Type Resolver - add resolve return type method

**Comment by JanMikes** [comment]
@TomasVotruba i found a bug:
```php
/**
 * @Symfony\Component\Routing\Annotation\Route(path="/events/<eventId>", methods={"GET"})
 */
```

`<eventId>`  should be `{eventId}`, symfony parameters format is like that 😄

**Comment by TomasVotruba** [comment]
Let me reged it :)

**Comment by TomasVotruba** [comment]
Issue here: https://github.com/rectorphp/rector/issues/1042

**Pull request merged**